src/Symfony/Component/ClassLoader/Tests/LegacyApcUniversalClassLoaderTest.php

```diff
@@ -21,7 +21,7 @@ class LegacyApcUniversalClassLoaderTest extends \PHPUnit_Framework_TestCase
     protected function setUp()
     {
         if (ini_get('apc.enabled') && ini_get('apc.enable_cli')) {
-            apcu_clear_cache('user');
+            apcu_clear_cache();
         } else {
             $this->markTestSkipped('APC is not enabled.');
```

Comment:
Hm, then the message should be changed [here](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/ClassLoader/ApcUniversalClassLoader.php#L80) and [here](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/ClassLoader/ApcUniversalClassLoader.php#L80) too. WDYT?

(And actually idk why we check here apc ini settings when `apcu_clear_cache` used)